proof/README.md

```diff
@@ -0,0 +1,31 @@
+# Software Analysis Workbench (SAW) Verification
+
+## Installation
+
+Download the SAW binary from the [SAW website](https://saw.galois.com).
+
+You will also need to install the Java Runtime Environment (this is required for running SAW with Java even if this feature is not used).
+
+You will also need to have Clang installed.
+
+You may also need to install [Yices](https://yices.csl.sri.com).
+
+### MacOS
+
+For mac users, make sure to install `llvm` from Homebrew.
+
+```bash
+
+ $> brew install llvm
+
+```
+
+## Generating the Bit-code
+
+SAW performs verification by first running symbolic execution on the C LLVM bit-code to generate a model of the computation. SAW then tries to prove this model equivalent to the corresponding model generated by the Cryptol specification using a backend SMT solver (default is Z3).
+
+So we need to generate the LLVM bit-code for the C code we want to verify. That is done by the following command:
+
+```bash
+clang -g -O1 -c -emit-llvm file.c -o file.bc
+```
```

Comment:
We should also add how to run the saw-script here.

Reply:
Ah, good point. I'll add another commit with this.